src/components/List.tsx

```diff
@@ -0,0 +1,66 @@
+import classNames from "classnames";
+import { nullValuesAsUndefined } from "null-as-undefined";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/core/_lists.scss";
+
+/**
+ * The types of {@link List}.
+ */
+export enum ListTypes {
+  Bullet = "bullet",
+  Number = "number"
+}
+
+/**
+ * The proptypes for {@link List}.
+ */
+export interface ListProps {
+  id?: string | null;
+  className?: string | null;
+  /**
+   * Defaults to an unordered list if no type is specified.
+   */
+  type?: ListTypes | null;
+  /**
+   * The items in the list. Can be anything from a string to another list
+   * component.
+   */
+  items: React.ReactNode[];
```

Comment:
I briefly wondered if this should actually be `children`, but I see the issues if they're text nodes. I wonder if we could use a dual API, so either `items` or `children` (but never both), to allow for more natural React using fragments or other components. Thoughts?

Reply:
I'm more for making it work better in a react way, I just don't know how we would get it to work for text nodes without introducing a new basic component that needs to be used as a child, also given that it's probably the primary user case for these lists. I'm not a fan of two ways of working with the same component personally.

Reply:
Fragments is how (e.g. `<>Item 1</><>Item 2</>`), but still, it's an extra layer of complexity for the user that's probably not great. Let's leave it as it is, then. Maybe add a non-docstring comment explaining why it's a list prop rather than children.